pg_cron--1.2--1.3.sql

```diff
@@ -0,0 +1,22 @@
+/* pg_cron--1.2--1.3.sql */
+
+CREATE SEQUENCE cron.runid_seq;
+CREATE TABLE cron.job_run_details (
+	jobid bigint,
+	runid bigint primary key default nextval('cron.runid_seq'),
+	job_pid integer,
+	database text,
+	username text,
+	command text,
+	status text,
+	return_message text,
+	start_time timestamptz,
+	end_time timestamptz
+);
+
+GRANT SELECT ON cron.job_run_details TO public;
+ALTER TABLE cron.job_run_details ENABLE ROW LEVEL SECURITY;
+CREATE POLICY cron_job_run_details_policy ON cron.job_run_details USING (username = current_user);
```

Comment:
I'm wondering whether users should also have delete permission on their own records.

Reply:
Thinking about it more. No, the administrator might want to decide on deletion policy.